docs/flow.md

```diff
@@ -0,0 +1,29 @@
+# git-flow
```

Comment:
This should be linked to from `docs/README.md` with a short explanation there, also.